chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -566,6 +602,37 @@ impl PartialWitnessActor {
         validators.sort();
         Ok(validators)
     }
+
+    /// Returns for each code-hash the contract code retrieved from the storage.
+    fn retrieve_contract_code<'a, I>(
```

Comment:
could you please simplify this, I don't think we need lifetime and generic parameter with the current usage

Reply:
inline again (reverted this change)